src/pages/HelpSupport/components/ContactPage/components/Table/Table.tsx

```diff
@@ -1,82 +1,116 @@
-      flexDirection: 'column',
-      justifyContent: 'space-around',
-      borderRadius: '10px',
-    }}
-  >
-    <div
-      className="flex justify-between black-border"
-      style={{
-        borderBottom: '1px solid black',
-        fontSize: '22px',
-        fontWeight: 'bold',
-        display: 'flex',
-        alignItems: 'center',
-        height: '63px',
-        padding: '0px 0px 0px 30px',
-      }}
-    >
-      <div>Frequently Asked Questions</div>
-    </div>
-    <div
-      className="flex justify-between black-border"
-      style={{
-        borderBottom: '1px solid black',
-        fontSize: '16px',
-        display: 'flex',
-        alignItems: 'center',
-        height: '63px',
-        padding: '0px 0px 0px 30px',
-      }}
-    >
-      <div>Will you be adding more templates later?</div>
-      <button type="button" className="mr-4">
-        ▼
-      </button>
-    </div>
-    <div
-      className="flex justify-between black-border"
-      style={{
-        borderBottom: '1px solid black',
-        fontSize: '16px',
-        display: 'flex',
-        alignItems: 'center',
-        height: '63px',
-        padding: '0px 0px 0px 30px',
-      }}
-    >
-      <div>Can I create custom report from scratch?</div>
-      <button type="button" className="mr-4">
-        ▼
-      </button>
-    </div>
-    <div
-      className="flex justify-between black-border"
-      style={{
-        fontSize: '16px',
-        display: 'flex',
-        alignItems: 'center',
-        height: '63px',
-        padding: '0px 0px 0px 30px',
-      }}
-    >
-      <div>How to remove your “Form created using Forms Ocean” tag?</div>
-      <button type="button" className="mr-4">
-        ▼
-      </button>
+interface TableProps {
+  onAnswerVisibilityChange: (isVisible: boolean) => void;
+}
+
+const Table: FC<TableProps> = ({ onAnswerVisibilityChange }: TableProps) => {
+  const [isAnswerVisible1, setIsAnswerVisible1] = useState<boolean>(false);
+  const [isAnswerVisible2, setIsAnswerVisible2] = useState<boolean>(false);
+  const [isAnswerVisible3, setIsAnswerVisible3] = useState<boolean>(false);
+
+  const toggleAnswerVisibility1 = (): void => {
+    setIsAnswerVisible1(!isAnswerVisible1);
+    onAnswerVisibilityChange(!isAnswerVisible1);
+  };
+
+  const toggleAnswerVisibility2 = (): void => {
+    setIsAnswerVisible2(!isAnswerVisible2);
+    onAnswerVisibilityChange(!isAnswerVisible2);
+  };
+
+  const toggleAnswerVisibility3 = (): void => {
+    setIsAnswerVisible3(!isAnswerVisible3);
+    onAnswerVisibilityChange(!isAnswerVisible3);
+  };
+
+  const handleAnswerVisibilityChange = (): JSX.Element => {
+    const visibleAnswers = [isAnswerVisible1, isAnswerVisible2, isAnswerVisible3].filter(Boolean);
+    const visibleAnswerCount = visibleAnswers.length;
+  
+    const newPageHeight = 900 + visibleAnswerCount * 100;
+  
+    return (
+      <div>
+        <p>New page height: {newPageHeight}</p>
+      </div>
+    );
+  };
```

Comment:
```suggestion js
const Table: FC<TableProps> = ({ onAnswerVisibilityChange }: TableProps) => {
  const [isAnswerVisible, setIsAnswerVisible] = useState<boolean[]>([false, false, false]);

  const toggleAnswerVisibility = (index: number): void => {
    const newVisibility = [...isAnswerVisible];
    newVisibility[index] = !newVisibility[index];
    setIsAnswerVisible(newVisibility);
    onAnswerVisibilityChange(newVisibility[index]);
  };

  const handleAnswerVisibilityChange = (): JSX.Element => {
    const visibleAnswerCount = isAnswerVisible.filter(Boolean).length;

    const newPageHeight = 900 + visibleAnswerCount * 100;

    return (
      <div>
        <p>New page height: {newPageHeight}</p>
      </div>
    );
  };
```

Reply:
The 3 Q&A can be structure simpler like this into an array map

Reply:
changed to array map，so cool，thank you